src/main/java/thePackmaster/cards/highenergypack/Food.java

```diff
@@ -21,6 +22,7 @@ public Food() {
     }
 
     public void use(AbstractPlayer p, AbstractMonster m) {
+        atb(new TalkAction(true, cardStrings.EXTENDED_DESCRIPTION[4], 1F, 2F));
         atb(new GainEnergyAction(magicNumber));
```

Comment:
Food is broken because it uses `magicNumber` here but never sets it. Right now it will give you -1 energy because of that.

Reply:
Interesting -- be aware that causes it to give you -1 energy if you try to test it by, for example, add it from the console.